lineage/figures/figure9.py

```diff
@@ -12,42 +12,22 @@
 from .figureCommon import getSetup, subplotLabel
 
 desired_num_states = np.arange(1, 5)
-# Lapatinib control pi and T for 2, 3, and 4 states:
-fpi_list = [[1.0], [0.66150779, 0.33849221], [6.94307126e-09, 6.99518861e-01, 3.00481132e-01], [6.67672155e-01, 3.32327845e-01, 2.86425040e-16, 4.61223911e-22]]
-
-fT_list = [[1.0], [[0.99277139, 0.00722861],
-                   [0.07170348, 0.92829652]], [[9.42386563e-01, 5.76134371e-02, 6.90068935e-28],
-                                               [4.77431313e-26, 7.85762403e-01, 2.14237597e-01],
-                                               [3.76423244e-01, 1.26698093e-02, 6.10906947e-01]], [[8.13843165e-01, 1.86156835e-01, 2.05422147e-13, 1.54246580e-15],
-                                                                                                   [4.65858601e-16, 1.45876129e-01, 1.13605784e-25, 8.54123871e-01],
-                                                                                                   [2.88389336e-01, 7.08946381e-01, 2.66398469e-03, 2.98246671e-07],
-                                                                                                   [1.19831901e-10, 2.22561316e-06, 9.72941919e-01, 2.70558550e-02]]]
-
 
 def makeFigure():
     """
     Makes figure 9.
     """
-    ax, f = getSetup((12, 6), (2, 4))
+    ax, f = getSetup((7, 3), (1, 2))
 
     data = [Lapatinib_Control[0:10], Lapt25uM[0:10], Lapt50uM[0:10], Lap250uM[0:10],
```

Comment:
When we removed small lineages, Gem30 will only have 13 lineages left. I thought it may be better that all have the same number of lineages. Wouldn't it be biased in some way if one condition has 100 lineages and the other has 13?

Reply:
We should use all the data available. If we have more data for one condition, it means we have more evidence for whatever number of states are present.

Reply:
Sure, will do.